### ISSUE TITLE
Accessibility: Content not included in landmarks - Exhibit Title

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#441]
**Description**
All content on a webpage should be inside landmarks. This allows screen readers to understand the purpose of different areas of the page at a glance based on the defined roles that these landmarks have.

**Solution**
Add `role="banner"` to  `<div class="container site-title-container">`

**Comment by mhk33** [comment]
Fixed in dev/3.0 upgrade
